src/middlewared/middlewared/etc_files/local/smb4.conf

```diff
@@ -0,0 +1,330 @@
+#
+# SMB.CONF(5)		The configuration file for the Samba suite 
+# $FreeBSD$
+#
+<%
+        import os
+        import sys
+        sys.path.extend([
+            '/usr/local/www',
+            '/usr/local/www/freenasUI'
+        ])
```

Comment:
The path should not be extended everytime, that will add up infinitely.

Reply:
This has been removed. We're not going to store parameters as sysctls, and so no need to fix.